src/spam-protection/payment-human-verification-handler.ts

```diff
@@ -0,0 +1,67 @@
+import { createScriptLoader } from '@bigcommerce/script-loader';
+import { switchMap, take } from 'rxjs/operators';
+
+import { RequestError } from '../common/error/errors';
+import { PaymentAdditionalAction } from '../payment';
+
+import { createSpamProtection } from './';
+import { CardingProtectionChallengeNotCompletedError, CardingProtectionFailedError, SpamProtectionChallengeNotCompletedError } from './errors';
+import GoogleRecaptcha from './google-recaptcha';
+
+export default class PaymentHumanVerificationHandler {
+    private _googleRecaptcha: GoogleRecaptcha;
+
+    constructor() {
+        this._googleRecaptcha = createSpamProtection(createScriptLoader());
```

Comment:
Can you create the Google ReCaptcha instance outside of this class and inject it into here instead?